lacework/resource_lacework_integration_gcr.go

```diff
@@ -156,6 +156,11 @@ func resourceLaceworkIntegrationGcr() *schema.Resource {
 				Optional: true,
 				Default:  5,
 			},
+			"non_os_package_eval": {
```

Comment:
let's call the new argument **non_os_package_support** similar like the UI shows it:
 
```suggestion
			"non_os_package_support": {
```

Reply:
Totally agree, updated 👍